csharp-tree-sitter.el

```diff
@@ -261,19 +263,20 @@
 (defgroup csharp-mode-indent nil "Indent lines using Tree-sitter as backend"
   :group 'tree-sitter)
 
-(defcustom csharp-mode-indent-offset 4
+(defcustom csharp-tree-sitter-indent-offset 4
```

Comment:
Won't removing the original defcustom break original csharp-mode?

Reply:
I tested and it was working normally, I don't think it was being used. However I think it was best to leave a `csharp-indent-offset` following convention of other major modes and setting an alias for tree-sitter for now. I made this change and nothing broke.

Reply:
Im 95% sure we dont need that in csharpmode, only in tree sitter version. Ill double check this evening :)

Reply:
https://github.com/emacs-csharp/csharp-mode/blob/master/csharp-mode.el#L242

We dont need an alias, just rename it